catalog/MaterialCatalog/MDCCatalogTiles.m

```diff
@@ -36,7 +36,7 @@
 #pragma clang diagnostic ignored "-Wassign-enum"
 #pragma clang diagnostic ignored "-Wconversion"
 
-void MDCCatalogDrawMDCLogoDark(CGRect frame, id<MDCColorScheme> colorScheme) {
+void MDCCatalogDrawMDCLogoDark(CGRect frame, id<MDCColorScheming> colorScheme) {
```

Comment:
Is this even used? I don't see any references in the method body

Reply:
Nope. But will remove in a separate PR.